contracts/tg4-group/src/contract.rs

```diff
@@ -0,0 +1,606 @@
+use crate::ContractError::Unauthorized;
```

Comment:
I assume this is mostly copied from cw-plus with `s/weight/points` renamed?

Reply:
That, plus introducing `TgradeMsg` binding. Plus adapting the hooks to tg-utils (based on the tg4-engagement impl)